src/main/kotlin/WorkerBuilder.kt

```diff
@@ -38,13 +44,14 @@ data class WorkerBuilder<T> private constructor(
     fun groupId(id: String) = option("group.id", id)
     fun option(k: String, v: Any) = options(mapOf(k to v))
     fun options(newOpts: Map<String, Any>) = copy(opts = opts + newOpts)
+    fun scope(scope: CoroutineScope): WorkerBuilder<T> = copy(scope = scope)
     fun setEngine(e: ConsumerActorFactory): WorkerBuilder<T> = copy(engine = e)
 
     fun install(i: WorkerInterceptor): WorkerBuilder<T> = copy(interceptors = interceptors.toMutableList().also{ it.add(i)})
 
     fun getInterceptors() = interceptors
 
-    fun start(scope: CoroutineScope = GlobalScope){
```

Comment:
This should never have been here in the first place. Nice fix.

Reply:
Yeah, that one is one me.